src/components/CreateProposal/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -157,7 +171,7 @@ class CreateSchemeRegistrarProposalContainer extends React.Component<IProps, ISt
                 upgradeController: false,
                 genericCall: false
               },
-              schemeToAdd: unregisteredSchemeAddresses.length > 0 ? unregisteredSchemeAddresses[0] : "Other",
+              // schemeToAdd: unregisteredSchemes.length > 0 ? unregisteredSchemes[0].address : "Other",
```

Comment:
dont we need this?

Reply:
well, aftere deciding to drop the idea of havinga picklist of unregisteredschemes, it does not make much sense anyremo..